src/Routers/ClassesRouter.js

```diff
@@ -114,7 +125,7 @@ export class ClassesRouter extends PromiseRouter {
       where,
       req.body,
       req.info.clientSDK
-    );
+    ).then(this.afterUpdate.bind(this, req));
```

Comment:
there may be a much more natural place to put this...I was hoping in the UserRouter, but the update handler there doesn't actually get called.  I don't understand exactly why that is, but the fact is that this is the handler that gets a called....

Reply:
I’m not sure why we need to duplicate the logic here. User handling should be done in one spot don’t you think?